paymaster-analysis/alchemy/alchemy-account.ts

```diff
@@ -0,0 +1,290 @@
+  throw new Error(
+    "Please populate .env file with demo Private Key. Recommended to not use your personal private key.",
+  );
+}
+
+const signer = privateKeyToAccount(privateKey as Hash);
+console.log("Signer Extracted from Private Key.");
+
+let publicClient;
+if (chain == "sepolia") {
+  publicClient = createPublicClient({
+    transport: http(rpcURL),
+    chain: sepolia,
+  });
+} else if (chain == "goerli") {
+  publicClient = createPublicClient({
+    transport: http(rpcURL),
+    chain: goerli,
+  });
+} else {
+  throw new Error(
+    "Current code only support Sepolia and Goerli. Please make required changes if you want to use custom network.",
+  );
+}
+
+// The console log in this function could be removed.
+const initCode = await getAccountInitCode({
+  owner: signer.address,
+  addModuleLibAddress:
+    SAFE_ADDRESSES_MAP[safeVersion][chainID].ADD_MODULES_LIB_ADDRESS,
+  safe4337ModuleAddress:
+    SAFE_ADDRESSES_MAP[safeVersion][chainID].SAFE_4337_MODULE_ADDRESS,
+  safeProxyFactoryAddress:
+    SAFE_ADDRESSES_MAP[safeVersion][chainID].SAFE_PROXY_FACTORY_ADDRESS,
+  safeSingletonAddress:
+    SAFE_ADDRESSES_MAP[safeVersion][chainID].SAFE_SINGLETON_ADDRESS,
+  saltNonce: saltNonce,
+  multiSendAddress,
+});
+console.log("\nInit Code Created.");
+
+const senderAddress = await getAccountAddress({
+  client: publicClient,
+  owner: signer.address,
+  addModuleLibAddress:
+    SAFE_ADDRESSES_MAP[safeVersion][chainID].ADD_MODULES_LIB_ADDRESS,
+  safe4337ModuleAddress:
+    SAFE_ADDRESSES_MAP[safeVersion][chainID].SAFE_4337_MODULE_ADDRESS,
+  safeProxyFactoryAddress:
+    SAFE_ADDRESSES_MAP[safeVersion][chainID].SAFE_PROXY_FACTORY_ADDRESS,
+  safeSingletonAddress:
+    SAFE_ADDRESSES_MAP[safeVersion][chainID].SAFE_SINGLETON_ADDRESS,
+  saltNonce: saltNonce,
+  multiSendAddress,
+});
+console.log("\nCounterfactual Sender Address Created:", senderAddress);
+
+// TODO This and in other files, this might be made easier with chain substituted at the right place.
+if (chain == "sepolia") {
+  console.log(
+    "Address Link: https://sepolia.etherscan.io/address/" + senderAddress,
+  );
+} else if (chain == "goerli") {
+  console.log(
+    "Address Link: https://goerli.etherscan.io/address/" + senderAddress,
+  );
+} else {
+  throw new Error(
+    "Current code only support Sepolia and Goerli. Please make required changes if you want to use custom network.",
+  );
+}
+
+const newNonce = await getAccountNonce(publicClient, {
+  entryPoint: ENTRY_POINT_ADDRESS,
+  sender: senderAddress,
+});
+console.log("\nNonce for the sender received from EntryPoint.");
+
+const contractCode = await publicClient.getBytecode({ address: senderAddress });
+
+const sponsoredUserOperation: UserOperation = {
+  sender: senderAddress,
+  nonce: newNonce,
+  initCode: contractCode ? "0x" : initCode,
+  callData: encodeCallData({
+    to: senderAddress,
+    data: "0xe75235b8", // getThreshold() of the Safe. TODO: Check if this could be removed.
```

Comment:
I'm pretty sure the Safe has a `fallback`/`receive` function that would accept empty calldata and value.

Reply:
Done 👍🏾